widgetssdk/src/main/java/com/glia/widgets/view/head/controller/ServiceChatBubbleController.java

```diff
@@ -107,6 +113,16 @@ public void onApplicationStop() {
         toggleChatBubbleServiceUseCase.execute(null);
     }
 
+    @Override
+    public void onChatHeadPositionChanged(int x, int y) {
+        chatHeadPositionHolder.setPosition(x, y);
+    }
+
+    @Override
+    public Pair<Integer, Integer> onChatHeadPositionRequested() {
```

Comment:
`onChatHeadPositionRequested` assumes it's more complex, like async call. but irl it's a getter `getChatHeadPoition`.